test/core/cont.wast

```diff
@@ -538,3 +538,476 @@
   (i32.const 0) (i32.const 1) (i32.const 2) (i32.const 3)
   (i32.const 4) (i32.const 5) (i32.const 6)
 )
+
+;; MVar implementation
+;; Scheduler
+
+(module $scheduler2
+  (type $proc (func))
+  (type $cont (cont $proc))
+  (type $susp_fn (func (param (ref null $cont))))
+
+  (event $yield (export "yield"))
+  (event $spawn (export "spawn") (param (ref $proc)))
+  (event $suspend (export "suspend") (param (ref $susp_fn)))
+  (event $resume (export "resume") (param (ref null $cont)))
+
+  ;; Table as simple queue (keeping it simple, no ring buffer)
+  (table $curr_proc 0 (ref null $cont))
+  (table $queue 0 (ref null $cont))
+
+  (global $qdelta i32 (i32.const 10))
+  (global $qback (mut i32) (i32.const 0))
+  (global $qfront (mut i32) (i32.const 0))
+  (global $curr_status (mut i32) (i32.const 0))
```

Comment:
It would be good to add a comment detailing the purpose of each of these variables.